### PR TITLE
eth_gasPrice

### PR DESCRIPTION
**Pull request by DockBoss**

### CODE REVIEW

File: json-rpc/eth_gasPrice.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# `Eth_gasPrice`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |Eth_gasPrice Must return price per gas in wei as type quantity|
+1.1 |Eth_gasPrice must return initial price per gas if there is no information from past transactions|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:6]: what is initial price per gas?

File: json-rpc/eth_gasPrice.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# `Eth_gasPrice`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |Eth_gasPrice Must return price per gas in wei as type quantity|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:5]: price per gas unit

Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:5]: eth <- lowercase

File: json-rpc/eth_gasPrice.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+1.1 |Eth_gasPrice must return initial price per gas if there is no information from past transactions|
+1.11|Eth_gasPrice must set initial price per gas before receiving any transactions|
+1.111|Eth_gasPrice's initial pric per gas May be set to 1 |
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:8]: 1 wei, not 1, best to define before use - initial price should be defined before it is referred to in the other points

File: json-rpc/eth_gasPrice.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# `Eth_gasPrice`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |Eth_gasPrice Must return price per gas in wei as type quantity|
+1.1 |Eth_gasPrice must return initial price per gas if there is no information from past transactions|
+1.11|Eth_gasPrice must set initial price per gas before receiving any transactions|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:7]: return, not set, wording is important

File: json-rpc/eth_gasPrice.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+1.111|Eth_gasPrice's initial pric per gas May be set to 1 |
+1.1111 |Eth_gasPrice must set gasPrice to the gasPrice of the first transaction right away if the initial price per gas is 1|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:9]: return not set
first transaction of what?

File: json-rpc/eth_gasPrice.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+1.2 |Eth_gasPrice may use a client defined method to estimate gasPrice if there is enough transaction data|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:10]: probably want to say MAY not may

File: json-rpc/eth_gasPrice.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+2 |Eth_gasPrice must consider a max gas price|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:11]: MUST not must?

Comment (tkstanczak) [reply json-rpc/eth_gasPrice.md:11]: what is max gas price?

File: json-rpc/eth_gasPrice.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+2.1 |Eth_gasPrice max gas price must be within the range of 0x1 to 0x7FFFFFFFFFFFFFFF|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:12]: what is max gas price, how is it set?

File: json-rpc/eth_gasPrice.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+2.2 |Eth_gasPrice must return the max gas price if transaction gasPrice is more than the max gas price|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:13]: which transaction?

File: json-rpc/eth_gasPrice.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+2.21 |Eth_gasPrice max gas price must be predefined or default|
```
Comment (tkstanczak) [inline json-rpc/eth_gasPrice.md:14]: what is the default

**Comment by SamWilsn** [comment]
Should be migrated to https://github.com/ethereum/eth1.0-apis

**Pull request closed without merging**